packages/datadog-plugin-graphql/src/index.js

```diff
@@ -16,7 +16,7 @@ function createWrapExecute (tracer, config, defaultFieldResolver) {
       const source = document && document._datadog_source
       const fieldResolver = args.fieldResolver || defaultFieldResolver
       const contextValue = args.contextValue = args.contextValue || {}
-      const operation = getOperation(document, args.operationName)
+      const operation = args.operation = getOperation(document, args.operationName)
```

Comment:
Why is this needed?

Reply:
It isn't. I assigned it to args, so I can pass it down to `getExecutionContext`. That way it wouldn't have to be recomputed. I removed this, since we'll want to use `graphql`'s native `buildExecutionContext` instead.